phpBB/phpbb/console/command/passwords/reset.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+namespace phpbb\console\command\passwords;
+
+use phpbb\console\command\command;
+use phpbb\db\driver\driver_interface;
+
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class reset extends command
```

Comment:
Well it would be over-written pretty quickly due to crons and the attachments one takes a while.